include/whole_body_state_rviz_plugin/WholeBodyStateDisplay.h

```diff
@@ -247,7 +247,6 @@ class WholeBodyStateDisplay : public rviz::MessageFilterDisplay<whole_body_state
   bool com_enable_;
   bool zmp_enable_;
   bool cop_enable_;
-  std::vector<Ogre::Vector3> cop_pos_;  //!< Stores locations of the foot center of pressures
```

Comment:
The changes permitted us to get rid of this member variable and rely on local variables, which is nice.